nms/app/packages/magmalte/app/views/subscriber/SubscriberOverview.js

```diff
@@ -144,92 +354,160 @@ function SubscriberTableRaw(props: WithAlert) {
           />
 
           {subscriberMap ? (
-            <ActionTable
-              data={Object.keys(subscriberMap).map((imsi: string) => {
-                const subscriberInfo = subscriberMap[imsi];
-                const metrics = subscriberMetrics?.[`${imsi}`];
-                return {
-                  name: subscriberInfo.name ?? imsi,
-                  imsi: imsi,
-                  service: subscriberInfo.lte.state,
-                  currentUsage: metrics?.currentUsage ?? '0',
-                  dailyAvg: metrics?.dailyAvg ?? '0',
-                  lastReportedTime: new Date(
-                    subscriberInfo.monitoring?.icmp?.last_reported_time ?? 0,
-                  ),
-                };
-              })}
-              columns={[
-                {title: 'Name', field: 'name'},
-                {
-                  title: 'IMSI',
-                  field: 'imsi',
-                  render: currRow => (
-                    <Link
-                      variant="body2"
-                      component="button"
-                      onClick={() =>
-                        history.push(relativeUrl('/' + currRow.imsi))
-                      }>
-                      {currRow.imsi}
-                    </Link>
-                  ),
-                },
-                {title: 'Service', field: 'service', width: 100},
-                {title: 'Current Usage', field: 'currentUsage', width: 175},
-                {title: 'Daily Average', field: 'dailyAvg', width: 175},
-                {
-                  title: 'Last Reported Time',
-                  field: 'lastReportedTime',
-                  type: 'datetime',
-                  width: 200,
-                },
-              ]}
-              handleCurrRow={(row: SubscriberRowType) => setCurrRow(row)}
-              menuItems={[
-                {
-                  name: 'View',
-                  handleFunc: () => {
-                    history.push(relativeUrl('/' + currRow.imsi));
+            <div>
+              <ActionTable
+                data={
+                  networkCtx.networkType !== FEG_LTE
```

Comment:
For a FEG_LTE network, the subscriber map itself might be empty, since the configuration isn't present. We have to make changes in subscriber context to ensure that we can collate this information and have a single subscriber state. We shouldn't have to do networkType checks here. Even for mocks, let's move the above mock state to subscriber state and ensure that we populate IMSI, MSISDN etc from subscriber state when possible.